src/client/modules/components/review-employee-row-day/template.html

```diff
@@ -22,7 +22,8 @@
     >
 
     {{#with utilization}}
-    <span class="utilization-type">{{ type.name }}</span>
+    <span class="utilization-type">{{ newType.name }}</span>
```

Comment:
Feel free to revert this change if I'm not understanding something... I wasn't sure why you were using `type.name` here instead of `newType.name`. It passes the UI tests so I assume it is okay?